vllm/assets/base.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
+from vllm.envs import VLLM_ASSETS_CACHE
+
+
+def get_cache_dir():
+    """Get the path to the cache for storing downloaded assets."""
+    path = Path(VLLM_ASSETS_CACHE)
```

Comment:
need to expanduser here?

Reply:
It is already done in `vllm.env` so no need to do it again here.